server/install.sh

```diff
@@ -12,7 +12,7 @@
 apt-get update && apt-get install -yq yasm
 mkdir -p $GOPATH/src/github.com/minio
 cd $GOPATH/src/github.com/minio
-git clone -b deis --single-branch https://github.com/deis/minio-src.git minio
+git clone -b master --single-branch https://github.com/minio/minio.git minio
```

Comment:
Minio people had said they are gonna make a release this weekend mostly... We can wait or we can rebase our Deis/minio-src branch to latest